client-app/modules/quotes/index.ts

```diff
@@ -36,17 +36,22 @@ const route: RouteRecordRaw = {
       ],
     },
   ],
-  beforeEnter(_to, _from, next) {
```

Comment:
@NaMax66 check this part please, I deleted that because I moved that condition into init function and seems it's not needed anymore